h2o-docs/src/product/data-science/generic.rst

```diff
@@ -0,0 +1,92 @@
+-  DRF (Distributd Random Forest)
+-  IRF (Isolation Random Forest)
+-  GLM (Generalized Linear Model)
+
+Importing a Generic model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As the model is generic, there are no hyperparameters and there is no real training phase. H2O imports the model and
+embraces it for the purpose of scoring. Information output about the model may be limited.
+
+.. example-code::
+   .. code-block:: r
+
+    data <- h2o.importFile(path = 'training_dataset.csv')
+    cols <- c("Some column", "Another column")
+    model.original <- h2o.glm(x=cols, y = "response", training_frame = data)    
+
+    path <- "/path/to/model/directory"
+    mojo.destination <- h2o.download_mojo(model = model.original, path = path)
+    model.generic <- h2o.genericModel(mojo.destination)
+
+    new_observations <- h2o.importFile(path = 'new_observations.csv')
+    h2o.predict(model.generic, new_observations)
+
+   .. code-block:: python
+
+    data = h2o.import_file(path='training_dataset.csv')
+    glm = H2OGeneralizedLinearEstimator()
+    glm.train(x = ["Some column", "Another column"], y = "response", training_frame=data)
+
+    path = '/path/to/model/directory/model.zip'
+    drf.download_mojo(path)
+
+    model = H2OGenericEstimator.from_file(path)
+    new_observations = h2o.import_file(path='new_observations.csv')
+    predictions = model.predict(new_observations)
+
+
+In Flow, click on 'Models' in the top menu and select 'Generic'. It is however required to import or upload the generic model (most likely a MOJO) beforehand. In order to to do, go to 'Data' in the top menu and select 'import files' or 'upload files'.
+
+Advanced Generic model initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is also possible to construct Generic model from already uploaded MOJO bytes. There is no need to import the MOJO
+every time a Generic model is created.
+
+.. example-code::
+   .. code-block:: r
+
+    data <- h2o.importFile(path = 'training_dataset.csv')
+    cols <- c("Some column", "Another column")
+    model.original <- h2o.glm(x=cols, y = "response", training_frame = data)    
+
+    path <- "/path/to/model/directory"
+    mojo.destination <- h2o.download_mojo(model = model.original, path = path)
+    
+    # Only import or upload MOJO model data, do not initialize the generic model yet
+    imported_mojo_key <- h2o.importFile(mojo.destination, parse = FALSE)
+    # Build the generic model later, when needed 
+    model.generic <- h2o.generic(model_key = imported_mojo_key)
+
+    new_observations <- h2o.importFile(path = 'new_observations.csv')
+    h2o.predict(model.generic, new_observations)
+
+   .. code-block:: python
+
+    data = h2o.import_file(path='training_dataset.csv')
+    glm = H2OGeneralizedLinearEstimator()
+    glm.train(x = ["Some column", "Another column"], y = "response", training_frame=data)
+
+    path = '/path/to/model/directory/model.zip'
+    drf.download_mojo(path)
+    
+    imported_mojo_key = h2o.lazy_import(file)
```

Comment:
same as above - make it easy for the users

Reply:
Same as above, read the previous part of the documentation. The convenience function comes first, then this.

Reply:
See the `Importing a Generic Model` section

Reply:
ok, thanks for explaining